### ISSUE TITLE
Custom type mapping for logical types

### ISSUE THREAD

**Issue by DaimonPl** [julianpeeters/avrohugger#96]
Iis there a way to specify custom type mapping for logical types?

For example I've got schema with `date` and `timestamp_ms` logical types which are currently generated to `java.time.LocalDate` and `java.time.LocalDateTime`

I'd like be able to customize that generation to `java.sql.Date` and `java.sql.Timestamp` (because of apache spark limitation)

**Comment by DaimonPl** [comment]
@julianpeeters any idea if it's supported? I went thorugh the code and it looks like java.time.* have no custom mapping available

**Comment by julianpeeters** [comment]
@DaimonPl sorry, that's not supported.

Seems like a good idea though, and I don't see why it wouldn't be possible. I may have some time later this week to take a closer look.

**Comment by DaimonPl** [comment]
@julianpeeters that would be great, thx :)

**Comment by DaimonPl** [comment]
@julianpeeters Hi, is there any way i could help with this?

**Comment by julianpeeters** [comment]
Sorry for the delay @DaimonPl, this should be available to try in avrhohugger version 1.0.0-RC12 and sbt-avrohugger version 2.0.0-RC12, at Sonatype now, and syncing with maven central shortly.

example use in sbt-avrohugger:
```scala
avroScalaCustomTypes in Compile := {
  avrohugger.format.Standard.defaultTypes.copy(
    date = avrohugger.types.JavaSqlDate,
    timestampMillis = avrohugger.types.JavaSqlTimestamp)
}
```

**Comment by DaimonPl** [comment]
Works great, thx! :)
